changelogs/fragments/10102-ini_file-fix-unmatched-whitespace-before-comment.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

This definitely is not a bugfix, but a feature.
```suggestion
minor_changes:
```

---

Fair enough, I'll update the description too from Bugfix Pull Request to Feature Pull Request.